src/components/UploadNoStyle/UploadNoStyle.tsx

```diff
@@ -67,8 +72,18 @@ export const UploadNoStyle = (props: UploadNoStyleProps) => {
         props.customRequest
           ? async req => {
```

Comment:
**suggestion (non-blocking)** might be slightly nicer to break this ternary into an `if (!props.customRequest) return undefined`